packages/cli-core/src/package-manager-controller/npm_package_manager_controller.ts

```diff
@@ -2,18 +2,18 @@ import { existsSync as _existsSync } from 'fs';
 import _fsp from 'fs/promises';
 import { execa as _execa } from 'execa';
 import * as _path from 'path';
-import { executeWithDebugLogger as _executeWithDebugLogger } from '../execute_with_logger.js';
-import { PackageManagerController } from './package_manager_controller.js';
+import { executeWithDebugLogger as _executeWithDebugLogger } from './execute_with_debugger_logger.js';
+import { PackageManagerControllerBase } from './package_manager_controller_base.js';
 
 /**
  * NpmPackageManagerController is an abstraction around npm commands that are needed to initialize a project and install dependencies
  */
-export class NpmPackageManagerController extends PackageManagerController {
+export class NpmPackageManagerController extends PackageManagerControllerBase {
   /**
    * constructor
    */
   constructor(
-    readonly projectRoot: string,
+    readonly projectRoot: string = './',
```

Comment:
I'd remove as many of these defaults as possible and make these params mandatory instead (in similar places too).
The project root should be resolved in single (or few) place and flow to components.

Reply:
also, if `./` is referring to `process.cwd()`, I would use that instead as it's more readable

Reply:
Done 2d94b47d71